### ISSUE TITLE
Maximum profit switch %

### ISSUE THREAD

**Issue by Steality** [tutulino/Megaminer#268]
Since some altcoin price sometimes are too good to be true to mine then leave the block orphaned, I think it will be good if maximum switching percentage added.

**Comment by densm** [comment]
There is @@PERCENTTOSWITCH option in config file

**Comment by Steality** [comment]
True, but it only check for which is more profitable by means of minimum checking. Ahashpool, hashrefinery recently known to be mining a wrong chain and ended up profitability above 600%~1000%, then leave the block orphaned, which waste of time and power. Setting maximum percenttoswitch is necessary for this pool. Lets say profit x17 suddenly is above 600%, MM will detect the profit is "too good to be true" by select the range 10%<x<600%.

**Comment by 1001rapid** [comment]
@Steality You just need to believe))

**Comment by eMTee72** [comment]
Agree... and hopefully that prevents scenarios like this one: 
![image](https://user-images.githubusercontent.com/32677058/36642662-133c22f6-1a43-11e8-8071-d363a4909225.png)

Although I wish it was true!  :-)

**Comment by or1g1na1** [comment]
I have seen this as well with zerg having crazy profitability on a coin but on other pools same coin is not even the most profitable.

**Comment by eMTee72** [comment]
IMHO ZPool and ZergPool are the ones that seem to pull additional traffic in with crazy rates.
